src/main/kotlin/com/exactpro/th2/check1/CollectorService.kt

```diff
@@ -140,11 +140,14 @@ class CollectorService(
     }
 
     private fun MessageFilter.toRootMessageFilter(): RootMessageFilter {
+        check(this.messageType.isNotEmpty()) { "Rule cannot be executed because the message filter does not contain 'message type'" }
```

Comment:
I would use `isNotBlank` instead of `isNotEmpty`. There is no point of having message with message type consists of whitespaces only